packages/core/core/src/factories/useCartFactory.ts

```diff
@@ -124,11 +121,9 @@ export const useCartFactory = <CART, CART_ITEM, PRODUCT, COUPON>(
       try {
         loading.value = true;
         const { updatedCart } = await factoryParams.removeCoupon({
-          currentCart: cart.value,
-          coupon: appliedCoupon.value
+          currentCart: cart.value
```

Comment:
how do you know which coupon you're removing?

Reply:
`getCouponFromCart` function will tell that.